superset/assets/package.json

```diff
@@ -80,6 +80,8 @@
     "mustache": "^2.2.1",
     "nvd3": "1.8.6",
     "parse-iso-duration": "^1.0.0",
+    "po2json": "^0.4.5",
```

Comment:
does that belong here? should be in dev if anywhere I think

Reply:
That was added by someone else and came with a merge conflict in `package.json`, let me remove it.